src/index.html

```diff
@@ -7,8 +7,99 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav__wrapper">
+        <a href="#" class="logo">
+          <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="logo__img"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
```

Comment:
```suggestion
              <a href="#" class="nav__link is-active">
                Apple
              </a>
```
You can add attributes in one line if their amount is less than 3